### PR TITLE
Add beginner tutorial

### PR DESCRIPTION
**Pull request by merelinguist**
### Type: feature 

Closes: n/a

### What are the changes and their implications? :gear:

Adds beginner tutorial for using Blitz

### Checklist

- [ ] Tests added for changes
- [ ] Any added terminal logging uses `packages/server/src/log.ts`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ Blitz **maximizes your productivity** both when starting an app and when scaling
 2. `blitz new myapp`
 3. [Read the Alpha User Guide](https://github.com/blitz-js/blitz/blob/canary/USER_GUIDE.md)
 
+or
+
+3. Start with the [Biltz Beginner Tutorial](https://github.com/blitz-js/blitz/blob/canary/TUTORIAL.md)
```
Comment (flybayer) [inline README.md:42]:
```suggestion
3. Start with the [Blitz Beginner Tutorial](https://github.com/blitz-js/blitz/blob/canary/TUTORIAL.md)
```

File: USER_GUIDE.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Before getting started, you should know **this is alpha software**. Blitz is inc
 
 If you have any issues at all, please [open an issue](https://github.com/blitz-js/blitz/issues/new/choose) or join the [Blitz slack](https://slack.blitzjs.com) and talk to us in the **#help** channel. If you get stuck and frustrated, please don't blame yourself. This user guide, and Blitz in general, is not yet fine-tuned for those with less experience. But eventually, it will be because this is very important to us.
 
+If you’re looking for a slower, more guided start to Blitz, read the [Biltz Beginner Tutorial](https://github.com/blitz-js/blitz/blob/canary/TUTORIAL.md).
```
Comment (flybayer) [inline USER_GUIDE.md:9]:
```suggestion
If you’re looking for a slower, more guided start to Blitz, read the [Blitz Beginner Tutorial](https://github.com/blitz-js/blitz/blob/canary/TUTORIAL.md).
```

**Comment by flybayer** [comment]
Fix typos and merge!

**Comment by merelinguist** [comment]
@all-contributors please add @merelinguist for doc

**Pull request merged**